test/network_policy_test.go

```diff
@@ -236,7 +332,7 @@ func validateNetworkPolicy(
 func validatePodEndpoint(
 	ctx goctx.Context, pod *corev1.Pod,
 	aeroCluster *asdbv1beta1.AerospikeCluster,
-	networkType asdbv1beta1.AerospikeNetworkType, isTLS bool, actual []string,
+	networkType asdbv1beta1.AerospikeNetworkType, isTLS bool, actual []string, configuredIP string,
 ) error {
 	podIP, hostInternalIP, hostExternalIP, _ := getIPs(ctx, pod)
```

Comment:
Any reason, why are we not fetching configuredIP from node like internal and externalIP, as we have already set in labels(if done successfully)

Reply:
Makes sense to return all IPs from `getIPs` func

Reply:
This func is common for `access` and `alternate-access`. So, if we fetch IP from getIPs() then we have to know which IP to match. At present we are already sending only that IP that needs to be matched.

Reply:
I think as of now it is fine, but we can write validatePodEndpoint in a better way in future, where we pass less arguments.